.github/workflows/codegen.yml

```diff
@@ -0,0 +1,37 @@
+name: Codegen
+
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+
+jobs:
+  codegen:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: ["android"] # "ios" will be added after codegen is fixed
```

Comment:
what is the context of this fix? is this blocked?

Reply:
Not blocked anymore, the fix was released in [RN 0.70.4](https://github.com/facebook/react-native/releases/tag/v0.70.4).